lib/deflation.cpp

```diff
@@ -125,33 +125,34 @@ namespace quda {
     std::vector<ColorSpinorField*> res;
     res.push_back(r);
 
-    for(int i = 0; i < nevs_to_print; i++)
-    {
-       zero(*r);
+    for (int i = 0; i < nevs_to_print; i++) {
+      zero(*r);
```

Comment:
not new in this pull but the blank lines confused me.

Reply:
Fixed in my next push